packages/strapi-admin/admin/src/containers/InstallPluginPage/saga.js

```diff
@@ -64,8 +64,14 @@ export function* pluginsGet() {
       },
     };
 
-    // Retrieve plugins list.
-    const availablePlugins = yield call(request, 'https://marketplace.strapi.io/plugins', opts);
+    let availablePlugins;
+
+    try {
+      // Retrieve plugins list.
+      availablePlugins = yield call(request, 'https://marketplace.strapi.io/plugins', opts);
```

Comment:
I would move `https://marketplace.strapi.io/plugins` to a const and reference it in `InstallPluginPage` and `ListPluginsPage`. Have this string hardcoded in both places is not a good practice.

Reply:
Sure, we will do this in another PR but you're totally right 👍